src/SegmentManager/SegmentManagerInterface.php

```diff
@@ -130,7 +131,7 @@ public function getSegmentBuilders();
      *    Therefore it's not a good idea to rely on the execution timestamp (time()) here.
      *    It's necessary to do this based on other criterias (for example activities).
      *  - If null was passed the timestamp will be set to null
-     * @param int|true|null $segmentApplicationCounter
+     * @param int|bool|null $segmentApplicationCounter
```

Comment:
I guess this should stay `true` as null is passed when not set.

Reply:
@dvesh3 Can be false. Is set to false here https://github.com/pimcore/customer-data-framework/blob/59753f6dd7e1b98f40c9d4350cfbbc792d557d49/src/ActionTrigger/Action/AddTrackedSegment.php#L86